conan/internal/api/new/qbs_lib.py

```diff
@@ -0,0 +1,46 @@
+from conan.internal.api.new.cmake_lib import source_cpp, source_h
+
+
+conanfile_sources = '''
+import os
+
+from conan import ConanFile, tools
+from conan.tools.qbs import Qbs
+from conan.tools.files import copy, collect_libs
+
+class {{package_name}}Recipe(ConanFile):
+    name = "{{name}}"
+    version = "{{version}}"
+
+    exports_sources = "*.cpp", "*.h", "*.qbs"
+    settings = "os", "compiler", "arch"
+
+    def build(self):
+        qbs = Qbs(self)
+        qbs.profile = ""
+        qbs.build()
+
+    def package(self):
+        qbs = Qbs(self)
+        qbs.profile = ""
+        qbs.install()
+
+    def package_info(self):
+        self.cpp_info.libs = collect_libs(self)
+'''
+
+qbs_file = '''
+    Library {
+        type: "staticlibrary"
```

Comment:
Is it possible to support a template that supports both static and shared lib, via ``shared`` option? If not, it is fine, but it would be good if possible.

Reply:
I wanted to do this in a separate PR as well as a test, but let's do this here